src/components/Comments/WriteComment.js

```diff
@@ -198,7 +199,7 @@ export const styles = StyleSheet.create({
     flex: 1
   },
   btn: (isDisableSubmit) => ({
-    backgroundColor: !isDisableSubmit ? COLORS.holyTosca : COLORS.concrete,
+    backgroundColor: !isDisableSubmit ? colors.bondi_blue : '#f2f2f2',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The background color of the button has been updated. Ensure this color change aligns with your design guidelines.
```diff
- backgroundColor: !isDisableSubmit ? COLORS.holyTosca : COLORS.concrete,
+ backgroundColor: !isDisableSubmit ? colors.bondi_blue : '#f2f2f2',
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->